types/dat-gui/index.d.ts

```diff
@@ -1,7 +1,10 @@
-// Type definitions for dat.GUI v0.6.1
+// Type definitions for dat.GUI v0.6.5
```

Comment:
`@types` packages are not supposed to have a minor version and that version is not supposed to start with `v`. This should be `dat.GUI 0.6` which is why the build is failing.